tests/test_db_containers.py

```diff
@@ -36,18 +37,42 @@ def test_docker_run_mariadb():
             assert row[0] == '10.2.9-MariaDB-10.2.9+maria~jessie'
 
 
-@pytest.mark.skip(reason="needs oracle client libraries unavailable on Travis")
-def test_docker_run_oracle():
-    oracledb_container = OracleDbContainer()
-    with oracledb_container as oracledb:
-        e = sqlalchemy.create_engine(oracledb.get_connection_url())
-        result = e.execute("select * from V$VERSION")
-        versions = {'Oracle Database 11g Express Edition Release 11.2.0.2.0 - 64bit Production',
-                    'PL/SQL Release 11.2.0.2.0 - Production',
-                    'CORE\t11.2.0.2.0\tProduction',
-                    'TNS for Linux: Version 11.2.0.2.0 - Production',
-                    'NLSRTL Version 11.2.0.2.0 - Production'}
-        assert {row[0] for row in result} == versions
+# @pytest.mark.skip(reason="needs oracle client libraries unavailable on Travis")
+# def test_docker_run_oracle():
+#     oracledb_container = OracleDbContainer()
+#     with oracledb_container as oracledb:
+#         e = sqlalchemy.create_engine(oracledb.get_connection_url())
+#         result = e.execute("select * from V$VERSION")
+#         versions = {'Oracle Database 11g Express Edition Release 11.2.0.2.0 - 64bit Production',
+#                     'PL/SQL Release 11.2.0.2.0 - Production',
+#                     'CORE\t11.2.0.2.0\tProduction',
+#                     'TNS for Linux: Version 11.2.0.2.0 - Production',
+#                     'NLSRTL Version 11.2.0.2.0 - Production'}
+#         assert {row[0] for row in result} == versions
+
+
+def test_docker_run_mongodb():
+    mongo_container = MongoDbContainer("mongo:latest")
+    with mongo_container as mongo:
+        db = mongo.get_connection_client().test
+        result = db.restaurants.insert_one(
+            {
+                "address": {
+                    "street": "2 Avenue",
+                    "zipcode": "10075",
+                    "building": "1480",
+                    "coord": [-73.9557413, 40.7720266]
+                },
+                "borough": "Manhattan",
+                "cuisine": "Italian",
+                "name": "Vella",
+                "restaurant_id": "41704620"
+            }
+        )
+        print(result.inserted_id)
+        cursor = db.restaurants.find({"borough": "Manhattan"})
+        for document in cursor:
+            print(document)
```

Comment:
Let's remove this print statement and instead validate the document returned by iterating over the cursor.

Reply:
done